data.md

```diff
@@ -517,9 +517,9 @@ The function interprets the range of bytes as little-endian.
     syntax Int ::= #getRange (ByteMap, Int , Int) [function, functional]
  // --------------------------------------------------------------------
     rule #getRange(BM, START, WIDTH) => 0
-      requires notBool (WIDTH >Int 0)  [concrete]
+      requires notBool (WIDTH >Int 0)
```

Comment:
these changes were needed to get it working with the Haskell backend (in an upcoming PR), so can't let this in.

Should be that if we go through a K submodule update, that the memory proofs work.

Reply:
Good catch. I'll wait for the submodule update. But I think the concrete-type memory proofs won't go through, because in those `#getRange` and `#setRange` won't get expanded into `#get`.

Reply:
Is it the case that `[concrete]` forces a function to apply only when the entire term is concrete? Or can the annotation be made more granular? In this case, it would be nice if we force only the width to be concrete (given that it would still work with the Haskell backend).

Reply:
You can have more granular annotation with the `#isConcrete` meta-function, which you can plaace in the side condition and call on individual variables.